Source/SwiftLintBuiltInRules/Rules/Lint/MultilineCallArgumentsRule.swift

```diff
@@ -0,0 +1,150 @@
+            Example("""
+                    foo(param1: 1)
+                    """,
+                    configuration: [
+                        "allows_single_line": false
+                    ]),
+            Example("""
+                    Enum.foo(param1: 1)
+                    """,
+                    configuration: [
+                        "allows_single_line": false
+                    ]),
+            Example("""
+                    Enum.foo(param1: 1, param2: 2, param3: 3)
+                    """,
+                    configuration: [
+                        "allows_single_line": true
+                    ]),
+            Example("""
+                    foo(
+                        param1: 1,
+                        param2: 2,
+                        param3: 3
+                    )
+                    """,
+                    configuration: [
+                        "allows_single_line": false
+                    ]),
+        ],
+        triggeringExamples: [
+            Example("↓foo(param1: 1, param2: false, param3: [])",
+                    configuration: [
+                        "max_number_of_single_line_parameters": 2
+                    ]),
+            Example("↓Enum.foo(param1: 1, param2: false, param3: [])",
+                    configuration: [
+                        "max_number_of_single_line_parameters": 2
+                    ]),
+            Example("""
+                    ↓foo(param1: 1, param2: false,
+                         param3: [])
+                    """, configuration: [
+                        "max_number_of_single_line_parameters": 3
+                    ]),
+            Example("""
+                    ↓Enum.foo(param1: 1, param2: false,
+                         param3: [])
+                    """, configuration: [
+                        "max_number_of_single_line_parameters": 3
+                    ]),
+            Example("""
+                    ↓foo(param1: 1, param2: false)
+                    """,
+                    configuration: [
+                        "allows_single_line": false
+                    ]),
+            Example("""
+                    ↓Enum.foo(param1: 1, param2: false)
+                    """,
+                    configuration: [
+                        "allows_single_line": false
+                    ]),
+        ]
+    )
+}
+
+private extension MultilineCallArgumentsRule {
+    final class Visitor: ViolationsSyntaxVisitor<ConfigurationType> {
+        override func visitPost(_ node: FunctionCallExprSyntax) {
+            guard node.arguments.isNotEmpty else { return }
+            guard node.trailingClosure == nil else { return }
+
+            if containsViolation(for: node.arguments) {
+                let anchor = node.calledExpression.positionAfterSkippingLeadingTrivia
+                violations.append(anchor)
+            }
+        }
+
+        private func containsViolation(for arguments: LabeledExprListSyntax) -> Bool {
```

Comment:
Let the other method extract the positions and remove this one.

Reply:
Done